src/engine/client/index.ts

```diff
@@ -130,11 +130,11 @@ export const createClient = ({
           })
           return result.data as EntryProps<any>
         },
-        publish: async ({ environment, entry, entryId }: PublishEntryParams) => {
+        publish: async ({ environment, entryVersion, entryId }: PublishEntryParams) => {
           count.cma++
-          return cmaClient.put(`${environment}/entries/${entryId}/published`, entry, {
+          return cmaClient.put(`${environment}/entries/${entryId}/published`, undefined, {
```

Comment:
Removing the entry, since the endpoint only needs the entryId and version

Reply:
Any idea why we had it there before?

Reply:
So this is an "either or" I assume, either the version number is deducted from the payload (whole entry) or from the header?

Reply:
We were passing the entry just to pick the version.
**cma.put** accepts the params below, and for publish we don't need to pass data 👍 
```url: string, data?: D, config?:```